### DISCUSSION TITLE
Difficulty Accessing Local Values.yaml File in argocd-repo-server Pod

### DISCUSSION THREAD

**Discussion by MrEsL** [argoproj/argo-cd#16306] [General]
Hello,

I'm currently facing an issue with accessing local values.yaml files from within the argocd-repo-server pod. Here's a bit of context:

I have an application that dynamically targets any repository URL and retrieves values from different repositories. In certain scenarios, I generate value files from Terraform, and these files are then mounted inside the repository pod.

However, the challenge arises when trying to access these values because they are not located within a Helm repository. The absence of a Helm repository seems to limit my ability to leverage these local value files from within the argocd-repo-server pod.

```
      helm:
        valueFiles:
        - /argocd-repo-server/localdir/values.yaml
```
Query:

Is there a viable solution or workaround that would allow me to utilize local value files effectively in this context? Any guidance or insights you could provide would be greatly appreciated.

**Comment by dudus46** [comment]
Hi,
We are currently facing the same issue with my team. Did you find anything so far ?

**Reply by MrEsL** [reply]
I tried multiple different options for example enabling soft links etc. but nothing worked. :( 

Would love to hear from you if you find anything

**Reply by Rider128** [reply]
We don't found how to import the value file locally in the git repository, so the solution for us is just to overwrite the needed values and use the defaults values from the project. 

Or another solution is to use this method https://github.com/argoproj/argocd-example-apps/blob/master/helm-dependency/README.md
And create a dependency to a custom values files. But I don't know if we do this how we can update everything easily.
